openslides/motion/forms.py

```diff
@@ -49,6 +49,11 @@ class BaseMotionForm(CleanHtmlFormMixin, CssClassMixin, forms.ModelForm):
     Attachments of the motion.
     """
 
+    tags = forms.ModelMultipleChoiceField(
+        queryset=Tag.objects.all(),
+        required=False,
+        label=ugettext_lazy('Tags'))
+
```

Comment:
Is this code block really required if you use `openslides.core.models import Tag`? For motion you use this in other way than in agenda and assignment.